css/styles.css

```diff
@@ -0,0 +1,157 @@
+* {
+	background-color: #FFFFFF;
+	font-family: sans-serif;
+}
+
+body {
+	width: 800px;
+	margin: auto;
+}
+
+nav {
+	height: 5px;
+	width: 800px;
```

Comment:
@humphriescarly One thing I'd like to point out is that you could have used `width: 100%` here instead of 800px. While both will work exactly the same for the page you've created, using 100% will be a bit more flexible if, say, later on down the road you wanted your page to be a bit wider. If you changed your page to be 1000px instead of 800px, you would also have to go and adjust _this_ css value as well. If you use a percentage, the css will adjust right along with you automatically :)